skycultures/arabic/dso_names.fab

```diff
@@ -0,0 +1,13 @@
+#NGC 869 |_("المعصم 1")
```

Comment:
Please use English names. If useful, add an English transliteration. 
Ah, those are commented away. Maybe delete (@alex-w ) ?

Reply:
This is to help me translate back to Arabic.
I added English names.